Uchu.Core/Extensions/RakConnectionExtensions.cs

```diff
@@ -16,7 +16,7 @@ public static void Send(this IRakConnection @this, ISerializable serializable)
             if (serializable == null)
                 throw new ArgumentNullException(nameof(serializable), 
                     ResourceStrings.RakConnectionExtensions_Send_StreamNullException);
-            Logger.Debug($"Sending {serializable}");
+            // Logger.Debug($"Sending {serializable}");
```

Comment:
Is there a reason why several debug statements were taken out? This applies to several files but I am only putting 1 comment for it.

Reply:
For racing it's constantly serializing people's positions which makes the logs kind of unreadable

Reply:
I see. In that case, I see 2 options: we remove the logging statements altogether, or we start using the `Trace` log level.